### ISSUE TITLE
Support for ICMP NAT port forward

### ISSUE THREAD

**Issue by samayer12** [pfsensible/core#28]
We're currently unable to create a a NAT port forward rule for ICMP over WAN. Is this a good idea in all "prod" networks? Perhaps not, but it's useful for test ranges and troubleshooting!

Why are we asking about NAT port forwards when NAT doesn't have a port? If the end goal is to provide the same amount of functionality via `pfsensible` as with the webUI, this may be worth the spare effort. This may already be covered by the firewall rule modules, if so I'll verify and close this issue.

Here's the desired state of a valid nat forward rule:

```
- Name: Create ICMP forward to some IP
  pfsensible.core.pfsense_nat_port_forward
    descr: ICMP to some IP
    interface: wan
    source: any
    protocol: icmp
    destination: “{{ some IP }}”
    target: “{{ a destination IP }}”
    associated_rule: associated
    state: “{{ portfwd_state }}”
  tags: my_tag
```

Here are the current errors I receive:
```
FAILED! => {“changed”: false, “msg”: “’Create ICMP forward to some IP’ on ‘wan’: “\”None\” is not a valid redirect target port. It must be a port alias or integer between 1 and 65535.”}
FAILED! => {“changed”: false, “msg”: “’Create ICMP forward to some IP’ on ‘wan’: “\”*\” is not a valid redirect target port. It must be a port alias or integer between 1 and 65535.”}
FAILED! => {“changed”: false, “msg”: “’Create ICMP forward to some IP’ on ‘wan’: “\”any\” is not a valid redirect target port. It must be a port alias or integer between 1 and 65535.”}
FAILED! => {“changed”: false, “msg”: “’Create ICMP forward to some IP’ on ‘wan’: You can’t use ports on protocols other than tcp, udp, tcp/udp”}
```

**Comment by opoplawski** [comment]
Could you give the changes in #49 a try?  Thanks.

**Comment by samayer12** [comment]
@opoplawski thanks for getting around to this! I've moved on from the team identified the initial use case so I'm unlikely to spend the time to verify it's what we needed back then.

That said, I reviewed the PR from #49 and it seems like that's what we were after when I opened this issue. The tests from `tests/unit/plugins/modules/test_pfsense_nat_port_forward.py` seems a lot like what we wanted at the time.
